addons/cppcheckdata.py

```diff
@@ -1516,6 +1516,14 @@ def ArgumentParser():
 def get_files(args):
     """Return dump_files, ctu_info_files"""
     all_files = args.dumpfile
+    for file in all_files:
+        if file.endswith('.dump'):
+            ctu_file = file.replace('.dump', '.ctu-info')
```

Comment:
I am skeptic about removing the ctu-info files. will the test case fail if you don't do this?